share/man/man1/intro.1

```diff
@@ -46,6 +49,17 @@ compilers and compiler tools,
 formatted output tools,
 and line printer commands.
 .Pp
+This section also contains most commands intended for daily user operation,
+installed with
+.Xr pkg 8 or
+.Xr ports 7 .
+Some of which include web browsers, office suites, media players,
+conferencing utilities, integrated development environments, audio
+and video processing suites, etc.
+.Fx
+has tens of thousands of third party programs available covering
```

Comment:
```diff
- FreeBSD has 
+ Ports collection has
```
but I'm not sure if it's good to add this line here. Maybe the user can already figure it out when they'll use ports tree?

Reply:
I rewrote it a few times. Maybe?
```
Tens of thousands of additional commands
are available to be installed with pkg(8), or
compiled with the ports(7) collection.
Some of which include * , etc.
```
Having tens of thousands of commands available is a relevant detail at the most surface level. This isn't mentioned on either pkg(8) or ports(7), and I'm not sure it should be. Those pages are on how to use those infrastructures. This page is the introduction to commands. The motd directs new users here, this directs them there. I agree the keyword "collection" is an important addition to ports the mention of ports. This made me realize that "compiled with" is also important, because the unfamiliar user should read the introduction and then look in the correct place for what they're trying to do. 

Thanks!

Reply:
Seems good to me, just...
```
< commands
> packages
```
e.g. libraries aren't related to commands, so "packages" sounds much okay

Reply:
Sounds better to us, but I think this document is for newbs. I'm concerned about forcing them to infer, in this page, that collections of installed files are called packages in this environment.

The term commands is also used exclusively throughout the page.